### PR TITLE
feature: link to groups, create chunk, upload file page from dashboard

### PR DESCRIPTION
**Pull request by vid277**
image

### CODE REVIEW

File: frontends/dashboard/src/components/Sidebar.tsx [hunk 1]
```diff
@@ -73,31 +91,59 @@ export const Sidebar = () => {
...
+                      <IoOpenOutline class="ml-1 inline-block h-4 w-4" />
+                    </a>
+                    <a
+                      href={`${searchUiURL}/upload${orgDatasetParams()}`}
+                      target="_blank"
+                      class="flex items-center hover:text-fuchsia-800"
+                    >
+                      Upload Data{" "}
+                      <IoOpenOutline class="ml-1 inline-block h-4 w-4" />
+                    </a>
```
Comment (skeptrunedev) [inline frontends/dashboard/src/components/Sidebar.tsx:128]: Change this to: 

- View Groups
- Create Chunk
- Upload File

**Review by skeptrunedev — CHANGES_REQUESTED** [review]

File: frontends/dashboard/src/components/Sidebar.tsx (hunk 1 above)
Comment (vid277) [reply frontends/dashboard/src/components/Sidebar.tsx:128]: fixed

File: frontends/dashboard/src/components/Sidebar.tsx [hunk 2]
```diff
@@ -73,31 +91,59 @@ export const Sidebar = () => {
...
-                  target="_blank"
-                  class="flex items-center"
+              <div class="flex flex-col items-center text-neutral-800">
+                <button
+                  onClick={() => setShowSubMenu(!showSubMenu())}
+                  class="flex items-center gap-2 hover:text-fuchsia-800"
                 >
-                  Search playground{" "}
+                  {showSubMenu() ? <FaSolidAngleDown /> : <FaSolidAngleRight />}
+                  <span>Search playground</span>
```
Comment (skeptrunedev) [inline frontends/dashboard/src/components/Sidebar.tsx:100]: this should still link out to the search page when clicked

File: frontends/dashboard/src/components/Sidebar.tsx [hunk 3]
```diff
@@ -73,31 +91,59 @@ export const Sidebar = () => {
...
-                  href={`${searchUiURL}${orgDatasetParams()}`}
-                  target="_blank"
-                  class="flex items-center"
+              <div class="flex flex-col items-center text-neutral-800">
+                <button
+                  onClick={() => setShowSubMenu(!showSubMenu())}
+                  class="flex items-center gap-2 hover:text-fuchsia-800"
                 >
-                  Search playground{" "}
+                  {showSubMenu() ? <FaSolidAngleDown /> : <FaSolidAngleRight />}
```
Comment (skeptrunedev) [inline frontends/dashboard/src/components/Sidebar.tsx:99]: The angle portion should expand and collapse the dropdown.

File: frontends/dashboard/src/components/Sidebar.tsx [hunk 4]
```diff
@@ -18,6 +26,16 @@ export const Sidebar = () => {
   const datasetContext = useContext(DatasetContext);
 
   const [showNewOrgModal, setShowNewOrgModal] = createSignal(false);
+  const [showSubMenu, setShowSubMenu] = createSignal(
+    localStorage.getItem("showSubMenu") === "true",
+  );
```
Comment (skeptrunedev) [inline frontends/dashboard/src/components/Sidebar.tsx:31]: this should default to open state

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
changes mentioned in other comments

**Review by densumesh — APPROVED** [review]

**Pull request merged**